Dockerfile

```diff
@@ -14,9 +47,11 @@ RUN pip install cython && \
     pip install --upgrade pip && \
     pip install --no-cache-dir -r requirements.txt
 
-EXPOSE 5000 8050
+#EXPOSE 1433 5000 8050
```

Comment:
Roland's production code (his docker-compose.yml file for production) may expect we have EXPOSE 5000 8050 in the Dockerfile. If this merge breaks the production webapp it's likely because of this. Just keep this in mind since removing all the ports from being exposed in the Dockerfile.